src/components/ConfigsListFilters/ConfigListFilterByGroupByDropdown.tsx

```diff
@@ -0,0 +1,66 @@
+import { ComponentProps, useState } from "react";
+import { ReactSelectDropdown } from "../ReactSelectDropdown";
+
+type ConfigGroupByDropdownProps = Omit<
+  ComponentProps<typeof ReactSelectDropdown>,
+  "name"
+>;
+
+export default function ConfigListFilterByGroupByDropdown({
```

Comment:
Can we move these to a filters sub folder and then just call them GroupByDropdown, ConfigTypeDropdown etc.